api/src/pcapi/routes/backoffice/accounts/blueprint.py

```diff
@@ -111,6 +111,48 @@ def _apply_search_filters(query: BaseQuery, search_filters: list[str]) -> BaseQu
     return query.filter(sa.or_(*or_filters))
 
 
+def is_beneficiary_anonymizable(user: users_models.User) -> bool:
+
+    if (
+        user.has_pro_role not in user.roles
+        and user.has_non_attached_pro_role not in user.roles
+        and user.has_admin_role not in user.roles
```

Comment:
Comprends-tu ce que tu as écrit ?
Tu testes donc `if True in user.roles` ou `if False in user.roles`...
Il suffisait pourtant de bien lire ma suggestion de mardi : 
```suggestion
        not user.has_pro_role
        and not user.has_non_attached_pro_role
        and not user.has_admin_role
```

Reply:
```
        user.has_pro_role
        and user.has_non_attached_pro_role
        and user.has_admin_role
```

FAIT